app/helpers/sessions_helper.rb

```diff
@@ -36,4 +36,17 @@ def log_out
     session.delete(:user_id)
     @current_user = nil
   end
+
+  def store_location
+    session[:forwarding_url] = request.original_url if request.get?
+  end
+
+  def redirect_back_or default
+    redirect_to(session[:forwarding_url] || default)
+    session.delete(:forwarding_url)
+  end
+
+  def hidden_with_admin
```

Comment:
```suggestion
  def hidden_with_admin?
```
các method return boolean nên đặt tên thêm ? ở sau

Reply:
Dạ. Sau em sẽ chú ý hơn